azurerm/resource_arm_netapp_pool.go

```diff
@@ -44,6 +44,7 @@ func resourceArmNetAppPool() *schema.Resource {
 			"account_name": {
 				Type:         schema.TypeString,
 				Required:     true,
+				ForceNew:     true,
```

Comment:
This is a breaking change? as well as the documentation should also be updated.

Reply:
NetApp Volume and NetApp Snapshot I haven' merged to master branch is associated with NetApp pool. So I think it has no meaning for only using NetApp Pool if NetApp Volume and NetApp Snapshot isn't ready. And it won't change the final result for this resource. So I think this isn't a breaking change. I've updated the documentation. Thanks.